openmdao/utils/scaffolding_templates/LinearSolver_template

```diff
@@ -0,0 +1,45 @@
+
+
+from openmdao.solvers.solver import LinearSolver
+
+
+class {class_name}(LinearSolver):
+    """
+    {class_name} solver.
+
+    Describe the solver here...
+    """
+
+    SOLVER = 'LN: ????'  # replace ???? with an abbreviated solver name
+
+    def __init__(self, **kwargs):
+        """
+        Initialize all attributes.
+
+        Parameters
+        ----------
+        **kwargs : dict
+            Options dictionary.
+        """
+        super({class_name}, self).__init__(**kwargs)
+
+        # initialize needed atributes here...
+
```

Comment:
I would add `_declare_options` and `_setup_solvers` here just to be consistent with the nl solver template.